docs/advanced.md

```diff
@@ -221,6 +221,28 @@ with httpx.Client(headers=headers) as client:
     ...
 ```
 
+## Monitoring download progress
+
+If you need to monitor download progress, you can stream with using `response.last_raw_chunk_size` property.
+
+For example, you can build a nice progress bar using the `tqdm` library:
```

Comment:
(Nit) Copy improvement.

(Super nit) Not sure we need the "nice" adjective so I replaced it with a more neutral phrasing below :)

```suggestion
For example, showing a progress bar using the [`tqdm`](https://github.com/tqdm/tqdm) library while a response is being downloaded could be done like this…
```